src/wagtail_localize_smartling/signal_handlers.py

```diff
@@ -0,0 +1,186 @@
+
+def create_landed_translation_task(
+    sender: Type["Job"],  # noqa: UP006
+    instance: "Job",
+    translation: "Translation",
+    **kwargs,
+):
+    if not smartling_settings.ADD_APPROVAL_TASK_TO_DASHBOARD:
+        logger.debug("Creation of a landed-translation task is disabled by settings")
+        return
+
+    LandedTranslationTask.objects.create_from_source_and_translation(  # pyright: ignore[reportAttributeAccessIssue]
+        source_object=instance.translation_source.get_source_instance(),
+        translated_locale=translation.target_locale,
+    )
+
+
+individual_translation_imported.connect(create_landed_translation_task, weak=False)
+
+
+def _close_translation_landed_task(instance):
+    c_type = ContentType.objects.get_for_model(instance)
+
+    # Try to find the task
+    tasks = LandedTranslationTask.objects.filter(
+        content_type=c_type,
+        object_id=instance.pk,
+    )
+    for t in tasks:
+        t.complete()
+
+
+def close_translation_landed_task_on_page_published(
+    sender: Type["Model"],  # noqa: UP006
+    instance: "Model",
+    **kwargs,
+):
+    # Both page_published and post_save send the core args, which
+    # is all we need
+
+    if not isinstance(instance, Page):
+        logger.debug(
+            f"{instance} is not a Page, so not trying to find a landed-translation task"
+        )
+        return
+    return _close_translation_landed_task(instance)
+
+
+page_published.connect(close_translation_landed_task_on_page_published, weak=False)
+
+
+def close_translation_landed_task_on_snippet_published(
+    sender: Type["Model"],  # noqa: UP006
+    instance: "Model",
+    **kwargs,
+):
+    # Both page_published and post_save send the core args, which
+    # is all we need
+    if not _model_is_registered_as_snippet(instance):
+        logger.debug(
+            f"{instance} is not a Snippet, so not trying to find a landed-translation task"
+        )
+        return
+    return _close_translation_landed_task(instance)
+
+
+post_save.connect(close_translation_landed_task_on_snippet_published, weak=False)
+
+
+def send_email_notification_upon_overall_translation_import(
+    sender: Type["Job"],  # noqa: UP006
+    instance: "Job",
+    translations_imported: list["Translation"],
+    **kwargs,
+):
+    """
+    Signal handler for receiving news that a translation has landed from
+    Smartling.
+
+    For now, sends a notification email to all Translation Approvers. TODO: make this a custom
+    notification option for all users in the Translation Approver group
+    """
+    if not smartling_settings.SEND_EMAIL_ON_TRANSLATION_IMPORT:
+        logger.info(
+            "Email notifications following translation import are disabled by settings"
+        )
+        return
+
+    ta_group = Group.objects.filter(
+        name=smartling_settings.TRANSLATION_APPROVER_GROUP_NAME
+    )
+    if not ta_group.exists():
+        logger.warning(
+            "Could not find the Translation Approvers "
```

Comment:
Should probably use the `smartling_settings.TRANSLATION_APPROVER_GROUP_NAME` here